src/java/org/apache/cassandra/repair/autorepair/RepairTokenRangeSplitter.java

```diff
@@ -486,7 +487,7 @@ static SizedRepairAssignment merge(List<SizedRepairAssignment> assignments)
     @VisibleForTesting
     protected List<SizedRepairAssignment> getRepairAssignmentsForTable(String keyspaceName, String tableName, Range<Token> tokenRange)
     {
-        List<SizeEstimate> sizeEstimates = getRangeSizeEstimate(keyspaceName, tableName, tokenRange);
+        List<AutoRepairUtils.SizeEstimate> sizeEstimates = AutoRepairUtils.getRangeSizeEstimate(repairType, keyspaceName, tableName, tokenRange);
```

Comment:
One thing we could maybe improve on here is that we already do this in `calcTotalBytesToBeRepaired` it looks like `KeyspaceRepairPlan` has `ksTablesEstimatedBytes`, I think we could just reuse that here to prevent touching the disk again to recalculate this data.

There is only one downside to this, in that since we're now doing the evaluation at the start, if the amount of data change between issuing repairs, this may skew things a bit, but I think its an acceptable tradeoff.